scripts/Jenkinsfile

```diff
@@ -4,6 +4,9 @@ pipeline {
            label 'qeff_node'
        }
    }
+   options {
+        disableConcurrentBuilds()
```

Comment:
in this case if there are multiple commits pushed it will execute it sequentially right?

Reply:
Yes